tests/run_tests.py

```diff
@@ -112,7 +120,7 @@ def test():
                         dest='cpp',
                         default=True,
```

Comment:
Just noticed that this argument has `default = True` and `action=store_true`, is it even possible to set it as "false"?